### ISSUE TITLE
Add a JSON file in config

### ISSUE THREAD

**Issue by AnniDineva** [eboye/laravel-nova-translation-editor#7]
How can I add a .json file in my trans_editor.php? 

<?php
return [
    'auth',
    'home',
    'countries',
    'validation',
    'invoices::invoice',
    'validation::inline'

];

**Comment by eboye** [comment]
It should be supported by only adding the name without [.json] extension to the array in the config.

I'm not using this library, so not really maintaining it anymore :|

But I have an project that would probably be using it in the near future. If so, I'll see if I can bring it up to speed to support Nova 4 etc.

**Comment by AnniDineva** [comment]
Yeah, I was try that, but It is no effect 😕. Maybe is important the path of files?

**Comment by eboye** [comment]
Note sure, it should be using the same scope as _('filename.somedeclaration') uses. So that filename if it's accessible from laravel scope it should be in this lib.
